src/read/coff/import.rs

```diff
@@ -0,0 +1,208 @@
+//! Support for reading short import files.
+//!
+//! These are used by some Windows linkers as a more compact way to describe
+//! dynamically imported symbols.
+
+use crate::read::{Architecture, Error, ReadError, ReadRef, Result};
+use crate::{pe, ByteString, Bytes, LittleEndian as LE};
+
+/// A Windows short form description of a symbol to import.
+/// Used in Windows import libraries. This is not an object file.
+#[derive(Debug, Clone)]
+pub struct MsImportFile<'data> {
+    header: &'data pe::ImportObjectHeader,
+    dll: ByteString<'data>,
+    symbol: ByteString<'data>,
+    kind: ImportType,
+    import: Option<ByteString<'data>>,
+}
+impl<'data> MsImportFile<'data> {
+    /// Parse it.
+    pub fn parse<R: ReadRef<'data>>(data: R) -> Result<Self> {
+        let mut offset = 0;
+        let header = pe::ImportObjectHeader::parse(data, &mut offset)?;
+
+        let mut strings = Bytes(
+            data.read_bytes(&mut offset, header.data_size() as u64)
+                .read_error("Not enough data")?,
```

Comment:
```suggestion
                .read_error("Invalid COFF import library data size")?,
```
Error messages need to be more specific about which file format the error occurred for, so need similar changes for other errors.